bin/install

```diff
@@ -146,7 +146,21 @@ os_based_configure_options() {
       configure_options="$configure_options --with-readline=$readline_path"
     fi
   else
+    local jpeg_path=$(locate libjpeg.so | head -n 1)
+    local libpng_path=$(locate libpng.so | head -n 1)
     configure_options="--with-openssl --with-curl --with-zlib --with-readline --with-gettext"
+
+    if [ "$jpeg_path" = "" ]; then
+      export ASDF_PKG_MISSING="$ASDF_PKG_MISSING jpeg"
```

Comment:
Those parts I just copied from the homebrew stuff.  I didn't check but assumed you used them somewhere else in the script.

We could certainly add error messages if it's missing.  It's not strictly a requirement to have the libraries, but as soon as you want to do anything with images you're likely going to need them.

Thanks for doing the original work!